libs/cardano-data/test/Test/Cardano/Data/OSet/Strict/Spec.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Test.Cardano.Data.OSet.Strict.Spec where
+
+import Data.OSet.Strict
+import Data.Sequence.Strict (StrictSeq, fromList)
+import Data.Set (Set, elems)
+import Test.Cardano.Data.Arbitrary ()
+import Test.Hspec
+import Test.Hspec.QuickCheck
+
+spec :: Spec
+spec =
+  describe "OSet.Strict" $ do
+    prop "membership checks work" $ \(s :: OSet Int) -> do
+      case uncons s of
+        Nothing -> pure ()
```

Comment:
Change to use `:<|:` pattern here instead, so we don't have to export `uncons` and `unsnoc`.